Docker.md

```diff
@@ -0,0 +1,121 @@
+Frank!Framework with Docker
+===========================
+
+Docker images are provided, suitable both for local and server use. Images are provided from the registry located at https://nexus.frankframework.org, the source is avalaible from the [docker-folder](docker/appserver/Tomcat) in this repository.
+
+# Contents
+
+- [General use](#General-use)
+  - [Local use](#Local-use)
+  - [Server use](#Server-use)
+- [Filesystem](#Filesystem)
+  - [Directories](#Directories)
+  - [Files](#Files)
+  - [Permissions](#Permissions)
+- [Logging](#Logging)
+- [Environment variables](#Environment-variables)
+- [Health and readiness](#Health-and-readiness)
+- [Considerations](#Considerations)
+  - [HTTPS and authentication](#HTTPS-and-authentication)
+  - [Secrets](#Secrets)
+  - [Non-root](#Non-root)
+
+
+General use
+===========
+The image contains an empty framework-instance that needs to be configured before use.
+
+Whether using the container locally or building your own image for use on servers, refer to [Filesystem](#Filesystem) information on which directories and files to mount or copy.
+
+For a list of available tags, see https://nexus.frankframework.org/#browse/search/docker.
+
+## Local use
+
+To run the image, run the following command, adding environment variables and mounts as needed:
+
+`docker run --publish <hostport>:8080 [-e <name>=<value>] [--mount type=bind,source=<source>,target=<target>] --name <name> nexus.frankframework.org/frank-framework[:<tag>]`
```

Comment:
Ik denk dat het goed is om een uitleg of voorbeeld te geven van alles wat je tussen '<' en '>' gespecificeerd hebt.